pyproject.toml

```diff
@@ -65,6 +65,7 @@ django-cors-headers = "^4.4.0"
 sentry-sdk = {extras = ["django"], version = "^2.13.0"}
 django-solo = "^2.3.0"
 pillow = "^10.4.0"
+google-generativeai = "^0.8.3"
```

Comment:
the `poetry.lock` file is missing (explains why the CI is failing), you need to run 
```
poetry add google-generativeai=0.8.3
```

Reply:
Or `poetry lock` to lock and updates `poetry.lock` file